_data/description.yml

```diff
@@ -4,6 +4,10 @@ aboutDescription: Learn more about the organization, including our main goals, c
 projectsTitle: Projects
 projectsDescription: Our projects cover a wide range of domains, from general functional programming to tooling.
 
+platformsTitle: Platforms
+platformsDescription: Checkout the Typelevel ecosystem by platform.
+jsDescription: Checkout Typelevel on Scala.js!
```

Comment:
I'm a little confused why this goes in a property -- is it used in more than one place?

But in general, all looks good to me!

Reply:
I was kind of just copying what I saw. But I do believe I'm going to move it out into the html because I want the text to have some hyperlinks. And I feel like the html is the more "obvious" place to find it.
AND... as you point out, it likely will not be used in more than one place :)